tests/helpers.py

```diff
@@ -156,13 +156,13 @@ def _do_execute_operation(self, operation: Operation) -> int:
 
         return rc
 
-    def _execute_install(self, operation: Operation) -> int:
+    def _execute_install(self, _: Operation) -> int:
```

Comment:
I don't like these changes. IMO if a method is overridden, the names of arguments shouldn't be changed.

Reply:
> IMO if a method is overridden, the names of arguments shouldn't be changed.

Glad to see, there's someone with the same opinion than me :smile:

Reply:
I generally agree with you, I mustn't have noticed that this was overridden! To be fair, there are already some instances on this around the poetry codebase..

Reply:
I must have changed it when I initially enabled all ARG rules, and then I reverted everything other than ARG001 because others were in fact too noisy when overriding methods